lisa/search_space.py

```diff
@@ -328,6 +328,9 @@ def remove(self, element: T) -> None:
         super().remove(element)
         self.items.remove(element)
 
+    def isunique(self, element: T) -> bool:
```

Comment:
if it's single item, it should follow other examples. Change the Setspace to the single item, instead of comparing it with a complex way.